libs/payments/stripe/src/lib/stripe.client.ts

```diff
@@ -166,6 +166,28 @@ export class StripeClient {
     return result as StripeResponse<StripeInvoice>;
   }
 
+  async invoicesUpdate(invoiceId: string, params?: Stripe.InvoiceUpdateParams) {
+    const result = await this.stripe.invoices.update(invoiceId, {
+      ...params,
+      expand: undefined,
+    });
+    return result as StripeResponse<StripeInvoice>;
+  }
+
+  async invoicesSetPayedOutOfBand(invoiceId: string) {
```

Comment:
This method should be called `invoicesPay` and should take the params for invoices.pay. The name `invoicesSetPayedOutOfBand` is a manager-level oriented method